huff_core/src/lib.rs

```diff
@@ -327,7 +338,12 @@ impl<'a> Compiler {
             tracing::info!(target: "core", "LOCALIZED IMPORTS {:?}", localized_imports);
         }
         let import_bufs: Vec<PathBuf> = Compiler::transform_paths(&localized_imports)?;
-        let mut file_sources: Vec<Arc<FileSource>> = Compiler::fetch_sources(&import_bufs);
+        let potentials: Result<Vec<Arc<FileSource>>, CompilerError> =
+            Compiler::fetch_sources(import_bufs).iter().cloned().collect();
```

Comment:
Use `.into_iter()` instead of `.iter().cloned()`

Reply:
Great call!